.github/workflows/new-pr-triage.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Move new PRs to New board in the Code Reviews Project
+
+on:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  automate-project-columns:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: alex-page/github-project-automation-plus@v0.2.3
+      with:
+        project: "Code Reviews"
+        column: New
+        repo-token: ${{ secrets.GH_TOKEN }}

```

Comment:
> I could probably switch it out for one with no hook or deployment permissions if people feel strongly about it.

I think it would be a very good idea to lock down this token as much as possible.